src/core/Akka.TestKit/TestKitBase_AwaitAssert.cs

```diff
@@ -33,7 +33,7 @@ public abstract partial class TestKitBase
         /// <param name="interval">The interval to wait between executing the assertion.</param>
         public void AwaitAssert(Action assertion, TimeSpan? duration=null, TimeSpan? interval=null)
         {
-            var intervalValue = interval.GetValueOrDefault(TimeSpan.FromMilliseconds(800));
```

Comment:
Wow, I had no idea the default value was this high....

Reply:
yeah, its a bit high